README.md

```diff
@@ -3,72 +3,96 @@
 Publisher is the primary content design app for GOV.UK. It provides the user interface for
 entering all the key editorial formats and an API so other apps (primarily frontend) can
 access that data for display. It is intended to work in partnership with Panopticon which
-manages metadata, slugs, titles, etc.
-
-## Running in development
-
-If you're just interested in running the Publisher locally, with a minimum of interaction
-with other apps, here's how.
-
-### Create a user
-
-    publisher$ rails console
-    >> User.create name: "Your name", email: "youremail@example.com", uid: Time.zone.now.to_i, version: 1
-
-### Run panopticon using rails s or similar
-
-    panopticon$ rails s -p 3001
-
-### Run the publisher app setting env variable to point at your panopticon instance
-
-    publisher$ PANOPTICON_URI="http://localhost:3001" bundle exec rails server
-
-## Local Transactions
-
-There is no UI or automated process for importing the source data for local transactions.
+manages document creation, metadata, slugs, titles, etc. It is sometimes referred to as "mainstream publisher".
+
+##Screenshots
+
+![alt tag](app/assets/images/publisher_document_screenshot.png)
```

Comment:
These images should go in the `doc/` directory so they don't become part of the application's asset pipeline